plugins/LayerTree.jsx

```diff
@@ -246,9 +249,14 @@ class LayerTree extends React.Component {
             "layertree-item-menubutton": true,
             "layertree-item-menubutton-active": this.state.activemenu === group.uuid
         });
+        const styleMenuClasses = classnames({
+            "layertree-item-menubutton": true,
+            "layertree-item-menubutton-active": this.state.activestyleselector === group.uuid
```

Comment:
I believe you can re-use `activestylemenu` instead of adding a new `activestyleselector` state variable.